src/main/java/org/github/flytreeleft/nexus3/keycloak/plugin/internal/KeycloakAdminClient.java

```diff
@@ -302,7 +307,25 @@ public AdapterConfig getConfig() {
 
     public synchronized Http getHttp() {
         if (this.http == null) {
-            HttpClient httpClient = HttpClients.createDefault();
+            CloseableHttpClient httpClient = null;
+            try {
+                httpClient = HttpClients.custom().
+                        setSSLHostnameVerifier(NoopHostnameVerifier.INSTANCE).
+                        setSSLContext(new SSLContextBuilder().loadTrustMaterial(null, new TrustStrategy()
+                        {
+                            @Override
+                            public boolean isTrusted(java.security.cert.X509Certificate[] x509Certificates, String s) throws java.security.cert.CertificateException {
+                                return true;
```

Comment:
we definitely do not want this. It will allow an attacker to hijack communication between Nexus and the Keycloak server. 

Instead, it could be documented that the user can add any CAs to a truststore by way of `keytool` and then set the `-Djavax.net.ssl.trustStore={{ nexus_jks_truststore }} -Djavax.net.ssl.trustStorePassword={{ nexus_jks_pass }}` java properties to actually trust the certificates.

Reply:
I agree with @maglo. @marctanguy can you revert this file?